Dockerfile

```diff
@@ -1,34 +1,60 @@
-FROM debian:stable-slim as fetcher
+# Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
 
-# TODO copy just the 'distribution' directory, not all source code
+FROM ubuntu:bionic as build
```

Comment:
is this going to change the resulting OS in the image ?

Reply:
No, it's just for the build

Reply:
So how the resulting OS will get set ? (or where from)

Reply:
From this command: `FROM tomcat:9-jre8`